source/localizable/routing/redirection.md

```diff
@@ -1,24 +1,44 @@
-Calling [`transitionTo()`][1] from a route or [`transitionToRoute()`][2] from a 
-controller will stop any transition currently in progress and start a new 
-one, functioning as a redirect. `transitionTo()` behaves exactly like the 
-[link-to](../../templates/links) helper.
+Sometimes you want to redirect a user to a different route than what they requested for.
+
+For example, if they're not logged in, you might want to prevent them from editing their profile, accessing private information,
+or checking out items in their shopping cart.
+Usually you want to redirect them to the login page, and after they have successfully logged in, redirect them back to the page the wanted to access.
+Or they're logged in already so you don't want them to be able to visit the login page again.
+
+There's many other reasons you probably want to have the last word on whether or not a user can access a certain route.
+Whatever the reason, Ember allows you that control with a combination of hooks and methods in your route.
+
+## The [`transitionTo()`][1] method
```

Comment:
I'm not a fan of these headers, as they veer into API documentation-land.
Can we switch it out for something more use case-based like "Transitioning Before the Model is Known" does?